tgstation.dmm-pal

```diff
@@ -0,0 +1 @@
+[Palette]
```

Comment:
What's this file for?

Reply:
Oh, it seems using the built-in map editor generates this file now. I should make a PR to include it in git-ignore.